From 3627a59c440d630f673a2d4ab7670d495f82b9c4 Mon Sep 17 00:00:00 2001
From: Greg Dietsche <Gregory.Dietsche@cuw.edu>
Date: Mon, 13 Jun 2011 09:40:38 -0500
Patch-mainline: 3.0-rc4
References: fate#310916
Git-commit: 96bf8bd1c953c3b9d89eac9f13dfdbf5e580060f
Subject: [PATCH 2511/2588] savage: remove unnecessary if statement

the code always returns ret regardless, so if(ret) check is unnecessary.

v2: fixed up the spelling.

Signed-off-by: Greg Dietsche <Gregory.Dietsche@cuw.edu>
Reviewed-by: Nicolas Kaiser <nikai@nikai.net>
Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit 96bf8bd1c953c3b9d89eac9f13dfdbf5e580060f)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/savage/savage_bci.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/savage/savage_bci.c b/drivers/gpu/drm/savage/savage_bci.c
index bf5f83e..cb1ee4e 100644
--- a/drivers/gpu/drm/savage/savage_bci.c
+++ b/drivers/gpu/drm/savage/savage_bci.c
@@ -647,9 +647,6 @@ int savage_driver_firstopen(struct drm_device *dev)
 	ret = drm_addmap(dev, aperture_base, SAVAGE_APERTURE_SIZE,
 			 _DRM_FRAME_BUFFER, _DRM_WRITE_COMBINING,
 			 &dev_priv->aperture);
-	if (ret)
-		return ret;
-
 	return ret;
 }
 
-- 
1.7.6

